docs/build-on-polkadot/parachains/asset-hub/register-a-local-asset.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Register a local asset
+description: Comprehensive guide to registering a local asset on the Asset Hub system parachain, including step-by-step instructions.
+---
+
+# Register a local asset on Asset Hub
+
+## Introduction
+As detailed in the [Asset Hub Overview](./overview.md#local-assets){target=_blank} page, Asset Hub accommodates two types of assets: local and foreign. Local assets are those that were created in Asset Hub and are identifiable by an integer ID. This guide will take you through the steps of registering a local asset on the Asset Hub parachain.
+
+## Prerequisites
+
+Before you begin, ensure you have access to the [Polkadot.js Apps](https://polkadot.js.org/apps/){target=_blank} interface and a Polkadot.js account with a balance of DOT or KSM.
+
+- For Polkadot Asset Hub, you would need a `deposit of 10 DOT` and around `0.201 DOT for the metadata`
+- For Kusama Asset Hub, the `deposit is 0.1 KSM` and around `0.000669 KSM for the metadata`
+
+You need to ensure that your Asset Hub account balance is a bit more than the sum of those two deposits, which should seamlessly account for the required deposits and transaction fees.
+
+## Environment setup
```

Comment:
This section is cool for testing, but maybe is not necessary here for explaining the process

Reply:
I will move it to the end of the article for now